src/mobile/src/main/java/tupperdate/android/ui/material/BrandedButton.kt

```diff
@@ -121,7 +125,33 @@ fun GradientButton(
         backgroundColor = Color.White,
         border = stroke,
         shape = RoundedCornerShape(50),
-        modifier = modifier.onPositioned { bounds = it.boundsInParent },
+        modifier = modifier.onGloballyPositioned { bounds = it.boundsInParent },
         content = content,
     )
-}
\ No newline at end of file
+}
+
+/**
+ * An alternative to [Button] that uses an animated gradient on its inner border. The gradient
+ * color scheme is Smurf- and Flamingo-based.
+ *
+ * @param value the string inside the button
+ * @param onClick the callback called when the button is clicked.
+ * @param modifier the modifier for the button.
+ */
+@Composable
+fun BrandedButton(
+    value: String,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    BrandedButton(
+        onClick = onClick,
+        modifier = modifier
+            .preferredHeight(56.dp),
+    ) {
+        Text(
+            text = (value).toUpperCase(Locale.getDefault()),
+            fontSize = 14.sp
```

Comment:
Not needed. Adding this alternative constructor / function for the `@Composable` is a nice idea !